tests/src/integration/apigateway/manifest.yml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: book club
+  version: 1.0
+  license: Apache-2.0
+  actions:
+      greeting:
+        version: 1.0
+        function: src/greeting.js
+        runtime: nodejs:6
+        inputs:
+          name: string
+          place: string
+        outputs:
+          payload: string
+  apis: # new top-level key for defining groups of named APIs
+    book-club: #api name
+      club: # shared base path
+        books:   #path
+           getBooks: get | post  #action name:verb
```

Comment:
@lzbj only one of get | put | delete appears.

Reply:
ok, got it ,thanks